### ISSUE TITLE
Not able to activate BOP on server.

### ISSUE THREAD

**Issue by BernardoEufrs** [Glitchfiend/BiomesOPlenty#1581]
## Bug Report

Cant make BOP work on my minecraft server

### What's the issue you encountered?

I know I have to create new worlds for it to work and I have to change the level-type to biomesoplenty in order for the minecraft_server to create a new world using BOP. But even doing this,  once I open the minecraft_server the new world is generated like a vanilla world and the changes made on server properties are reverted.

### How can the issue be reproduced?

Putting biomesoplenty mod on mods folder, changing level-type to biomesoplenty, opening minecraft_server to generate a new world.

### Logs

No logs, the game doesnt crash or anything.

### Mod Version
BiomesOPlenty-1.15.2-10.0.0.345-universal
forge-1.15.2-31.1.39
minecraft_server.1.15.2

**Comment by Forstride** [comment]
Are you *sure* you're actually using the Forge jar for the server?

**Comment by BernardoEufrs** [comment]
@Forstride absolutely

**Comment by Forstride** [comment]
Others have reported that it's working with the latest version of Forge, so it's really making me think it's something on your end.  I'll have to test it myself at some point though

**Comment by BernardoEufrs** [comment]
I'll make everything again right now and list every single one of my steps.

Downloading Recommended Forge Installer 1.15.2 - 31.1.0
Installing server to an empty folder
Sucessfully downloaded minecraft server and installed 1.15.2-forge-31.1.0
Folder now have some files
Executing "forge-1.15.2-31.1.0" in the folder to create the rest of the files
Mods folder and server properties created as intended.
Downloading and pasting "BiomesOPlenty-1.15.2-10.0.0.345-universal" into the server mods folder
Changing server.properties server-type to "biomesoplenty" and setting the ip
Changing eula to true
Executing "minecraft_server.1.15.2"
World generated as a vanilla world, level type changed back to default by "minecraft_server.1.15.2"

I don't think I'm doing something wrong here but let me know.

**Comment by Forstride** [comment]
>31.1.0

You said you were using 31.1.39 in your issue post.  The issue was fixed in Forge 31.1.29, so if you're using 31.1.0, then yeah, it's still gonna be broken...

**Comment by SexyFutanahri** [comment]
I am also having the same issue too, I tried downloading the new version of forge but when I try to place a block from the mod, it disappears as soon as I place it. Its like I place the block and then it just doesn't render BUT THE WEIRD THING IS, in my crafting guide, there are blocks that says, "biomes o plenty" so its definitely in the server but I can't do anything with it. I am using the latest version of everything and its still bugs out. I don't know what to do :((

**Comment by Forstride** [comment]
If you can see blocks in your creative inventory but can't place them, then the mod isn't loaded on the server (Or the other way around, not entirely sure).

**Comment by SexyFutanahri** [comment]
Can you list everything I should do? Like the mod isnt working or generating anything :((

**Comment by ellbristow** [comment]
I'm having the same issue with a new server.

Forge 1.15.2-31.1.43
BoP 1.15.2-10.0.0.345-universal

level-type=biomesoplenty

The world is a vanilla world

**Comment by SexyFutanahri** [comment]
> EDIT: Mine is now working

What did you do?

**Comment by ellbristow** [comment]
> > EDIT: Mine is now working
> 
> What did you do?

In my case, I stopped the server, made sure the mod was in place, and the config was set to level-type=biomesoplenty , then deleted the world, and started the server again

**Comment by SexyFutanahri** [comment]
Mine is still not working, I dont know what I need to do, I tried everything and it doesnt work on a server. Can someone lay the steps for me to follow with each version so I can make it work?

**Comment by ellbristow** [comment]
Just did it again and got a vanilla world again. generating locally with the same mods and config works, but server generation seems to be vanilla

I see this line in the log every time the server starts:

[13Apr2020 10:06:13.852] [modloading-worker-1/INFO] [biomesoplenty/]: Injecting biomesoplenty world type into server.properties. Original value: default

however in server.properties the level-type is set to bimoesoplenty

**Comment by Forstride** [comment]
The injecting line is from the previous method we had to use to allow servers to work.  The typo is entirely on your end, and why it's not working...

**Comment by ellbristow** [comment]
sorry, the typo is only here, in the config its correct

biomesoplenty

I just tried a fresh server with ONLY BoP installed, with the level-type set correctly and got vanilla gen

forge version 31.1.43
BoP version 1.15.2-10.0.0.345

**Comment by Forstride** [comment]
And you're absolutely 100% sure it's not just you spawning in a vanilla biome by chance?  Have you tried doing /bop tpbiome *biome_name*?  For example, /bop tpbiome biomesoplenty:coniferous_forest

**Comment by ellbristow** [comment]
I spawned in plains surrounded by two other vanilla biomes. I will try the TP though

**Comment by ellbristow** [comment]
Just deleted the world and regernerated and this time is IS a BoP world... very confused

**Comment by Forstride** [comment]
Vanilla biomes still exist in BOP worlds.  Either Forge's server properties fix isn't working all the time, or you're just thinking it's a BOP world

**Comment by chriellex** [comment]
Well. It looks like it's working now, we generated a new server.properties and it works. Not sure y it didn't before

**Comment by Buzingo** [comment]
I am facing a similar issue to this. I tried generating a new World on a MC 1.15.2 Server.
 I changed generator-settings= and level-type to BIOMESOP. Result: Server Crashes(See logs attached). 
Then I tried to only change level-type. This does not result in a crash but causes the same issue as written above. 
A pure vanilla world is generated. 

I am using forge-1.15.2-31.1.44. and BiomesOPlenty-1.15.2-10.0.0.353-universal. 

[crash-2020-04-13_18.07.33-server.txt](https://github.com/Glitchfiend/BiomesOPlenty/files/4470435/crash-2020-04-13_18.07.33-server.txt)

**Comment by zISKo666** [comment]
Hello,
I've tried to install a 1.15.2 server with BoP but it always crashes.
I use the latest forge installer 31.1.46 and the latest BoP 10.0.0.353.
No other mods are installed.
If I remove the **_biomesoplenty_** from the generator-settings= line than the server is starting and generating a new world.
Below you'll find the server.properties, crash-report and the latest.log.

server.properties:
```
#Minecraft server properties
#Sat Apr 18 16:39:59 CEST 2020
allow-flight=true
allow-nether=true
broadcast-console-to-ops=true
broadcast-rcon-to-ops=true
difficulty=easy
enable-command-block=false
enable-query=false
enable-rcon=false
enforce-whitelist=false
force-gamemode=false
function-permission-level=2
gamemode=survival
generate-structures=true
generator-settings=biomesoplenty
hardcore=false
level-name=world
level-seed=
level-type=default
max-build-height=256
max-players=4
max-tick-time=60000
max-world-size=29999984
motd=A Minecraft Server
network-compression-threshold=256
online-mode=true
op-permission-level=4
player-idle-timeout=0
prevent-proxy-connections=false
pvp=true
query.port=25565
rcon.password=
rcon.port=25575
resource-pack=
resource-pack-sha1=
server-ip=
server-port=25565
snooper-enabled=true
spawn-animals=true
spawn-monsters=true
spawn-npcs=true
spawn-protection=16
use-native-transport=true
view-distance=10
white-list=true
```

crash-report:
```
---- Minecraft Crash Report ----

// Hey, that tickles! Hehehe!

Time: 18.04.20 16:40
Description: Exception in server tick loop

com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $
	at net.minecraft.util.JSONUtils.func_188173_a(SourceFile:496) ~[?:?] {re:classloading}
	at net.minecraft.util.JSONUtils.func_212744_a(SourceFile:547) ~[?:?] {re:classloading}
	at net.minecraft.util.JSONUtils.func_212746_a(SourceFile:543) ~[?:?] {re:classloading}
	at net.minecraft.util.JSONUtils.func_212745_a(SourceFile:551) ~[?:?] {re:classloading}
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:209) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:613) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242] {}
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1559) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1401) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:593) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:425) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:716) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:714) ~[server-1.15.2-extra.jar:?] {}
	at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:910) ~[server-1.15.2-extra.jar:?] {}
	at net.minecraft.util.JSONUtils.func_188173_a(SourceFile:494) ~[?:?] {re:classloading}
	... 6 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.15.2
	Minecraft Version ID: 1.15.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_242, Amazon.com Inc.
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Amazon.com Inc.
	Memory: 6264193032 bytes (5974 MB) / 10737418240 bytes (10240 MB) up to 10737418240 bytes (10240 MB)
	CPUs: 8
	JVM Flags: 12 total; -Xms10G -Xmx10G -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -XX:MaxGCPauseMillis=100 -XX:+DisableExplicitGC -XX:TargetSurvivorRatio=90 -XX:G1NewSizePercent=50 -XX:G1MaxNewSizePercent=80 -XX:G1MixedGCLiveThresholdPercent=35 -XX:+AlwaysPreTouch -XX:+ParallelRefProcEnabled
	ModLauncher: 5.0.0-milestone.4+67+b1a340b
	ModLauncher launch target: fmlserver
	ModLauncher naming: srg
	ModLauncher services: 
		/eventbus-2.0.0-milestone.1-service.jar eventbus PLUGINSERVICE 
		/forge-1.15.2-31.1.46.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.15.2-31.1.46.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.0.4-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.15.2-31.1.46.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.15.2-31.1.46.jar runtimedistcleaner PLUGINSERVICE 
		/forge-1.15.2-31.1.46.jar fml TRANSFORMATIONSERVICE 
	FML: 31.1
	Forge: net.minecraftforge:31.1.46
	FML Language Providers: 
		javafml@31.1
		minecraft@1
	Mod List: 
		forge-1.15.2-31.1.46-server.jar Minecraft {minecraft@1.15.2 DONE}
		forge-1.15.2-31.1.46-universal.jar Forge {forge@31.1.46 DONE}
		BiomesOPlenty-1.15.2-10.0.0.353.jar Biomes O' Plenty {biomesoplenty@1.15.2-10.0.0.353 DONE}
	Player Count: 0 / 4; []
	Data Packs: 
	Is Modded: Definitely; Server brand changed to 'forge'
	Type: Dedicated Server (map_server.txt)
```

latest.log:
```
[18Apr2020 16:39:41.461] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--gameDir, ., --launchTarget, fmlserver, --fml.forgeVersion, 31.1.46, --fml.mcpVersion, 20200414.225201, --fml.mcVersion, 1.15.2, --fml.forgeGroup, net.minecraftforge, nogui]
[18Apr2020 16:39:41.461] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 5.0.0-milestone.4+67+b1a340b starting: java version 1.8.0_242 by Amazon.com Inc.
[18Apr2020 16:39:41.836] [main/INFO] [net.minecraftforge.fml.loading.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[18Apr2020 16:39:42.851] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'fmlserver' with arguments [--gameDir, ., nogui]
[18Apr2020 16:39:54.570] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[18Apr2020 16:39:54.570] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[18Apr2020 16:39:54.601] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[18Apr2020 16:39:54.617] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[18Apr2020 16:39:54.648] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[18Apr2020 16:39:54.888] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting minecraft server version 1.15.2
[18Apr2020 16:39:55.832] [modloading-worker-1/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 31.1.46, for MC 1.15.2 with MCP 20200414.225201
[18Apr2020 16:39:55.837] [modloading-worker-1/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v31.1.46 Initialized
[18Apr2020 16:39:58.486] [Server thread/WARN] [net.minecraft.entity.EntityType/]: No data fixer registered for entity biomesoplenty:boat_bop
[18Apr2020 16:39:58.726] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file C:\Minecraft-Server\Forge Test 1.15.2\config\biomesoplenty\generation.toml is not correct. Correcting
[18Apr2020 16:39:58.729] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld was corrected from null to SimpleCommentedConfig:{}
[18Apr2020 16:39:58.731] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld.biome_size was corrected from null to MEDIUM
[18Apr2020 16:39:58.732] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld.river_size was corrected from null to MEDIUM
[18Apr2020 16:39:58.733] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld.temperature_variation_scheme was corrected from null to MEDIUM_ZONES
[18Apr2020 16:39:58.734] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld.rainfall_variation_scheme was corrected from null to MEDIUM_ZONES
[18Apr2020 16:39:58.735] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key overworld.enhance_vanilla_biomes was corrected from null to true
[18Apr2020 16:39:58.755] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file C:\Minecraft-Server\Forge Test 1.15.2\config\biomesoplenty\server.toml is not correct. Correcting
[18Apr2020 16:39:58.757] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key biomes was corrected from null to SimpleCommentedConfig:{}
[18Apr2020 16:39:58.758] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key biomes.use_world_type was corrected from null to true
[18Apr2020 16:39:59.347] [modloading-worker-4/INFO] [biomesoplenty/]: Injecting biomesoplenty world type into server.properties. Original value: default
[18Apr2020 16:39:59.351] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/maven/net/minecraftforge/forge/promotions_slim.json
[18Apr2020 16:39:59.371] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Loading properties
[18Apr2020 16:39:59.415] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Default game type: SURVIVAL
[18Apr2020 16:39:59.416] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Generating keypair
[18Apr2020 16:39:59.672] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting Minecraft server on *:25565
[18Apr2020 16:39:59.687] [Server thread/INFO] [net.minecraft.network.NetworkSystem/]: Using default channel type
[18Apr2020 16:40:00.176] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: AHEAD Current: 31.1.46 Target: null
[18Apr2020 16:40:01.522] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file .\world\serverconfig\forge-server.toml is not correct. Correcting
[18Apr2020 16:40:01.524] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server was corrected from null to SimpleCommentedConfig:{}
[18Apr2020 16:40:01.529] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.removeErroringEntities was corrected from null to false
[18Apr2020 16:40:01.531] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.removeErroringTileEntities was corrected from null to false
[18Apr2020 16:40:01.532] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.fullBoundingBoxLadders was corrected from null to false
[18Apr2020 16:40:01.533] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.zombieBaseSummonChance was corrected from null to 0.1
[18Apr2020 16:40:01.535] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.zombieBabyChance was corrected from null to 0.05
[18Apr2020 16:40:01.535] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.logCascadingWorldGeneration was corrected from null to true
[18Apr2020 16:40:01.536] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.fixVanillaCascading was corrected from null to false
[18Apr2020 16:40:01.536] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.dimensionUnloadQueueDelay was corrected from null to 0
[18Apr2020 16:40:01.537] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.clumpingThreshold was corrected from null to 64
[18Apr2020 16:40:01.538] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.treatEmptyTagsAsAir was corrected from null to false
[18Apr2020 16:40:01.631] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Preparing level "world"
[18Apr2020 16:40:01.632] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $
	at net.minecraft.util.JSONUtils.func_188173_a(SourceFile:496) ~[?:?]
	at net.minecraft.util.JSONUtils.func_212744_a(SourceFile:547) ~[?:?]
	at net.minecraft.util.JSONUtils.func_212746_a(SourceFile:543) ~[?:?]
	at net.minecraft.util.JSONUtils.func_212745_a(SourceFile:551) ~[?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:209) ~[?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:613) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1559) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1401) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:593) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:425) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:716) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:714) ~[server-1.15.2-extra.jar:?]
	at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:910) ~[server-1.15.2-extra.jar:?]
	at net.minecraft.util.JSONUtils.func_188173_a(SourceFile:494) ~[?:?]
	... 6 more
[18Apr2020 16:40:01.649] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: This crash report has been saved to: C:\Minecraft-Server\Forge Test 1.15.2\.\crash-reports\crash-2020-04-18_16.40.01-server.txt
[18Apr2020 16:40:01.664] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping server
[18Apr2020 16:40:01.680] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving players
[18Apr2020 16:40:01.680] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving worlds
[18Apr2020 16:40:01.687] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Exception stopping the server
java.lang.IllegalArgumentException: Can not hotload overworld. This must be loaded at all times by main Server.
	at org.apache.commons.lang3.Validate.isTrue(Validate.java:158) ~[server-1.15.2-extra.jar:?]
	at net.minecraftforge.common.DimensionManager.initWorld(DimensionManager.java:229) ~[?:?]
	at net.minecraftforge.common.DimensionManager.getWorld(DimensionManager.java:198) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71218_a(MinecraftServer.java:1004) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_213211_a(MinecraftServer.java:536) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71260_j(MinecraftServer.java:568) ~[?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71260_j(DedicatedServer.java:550) ~[?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:676) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```
